content/en/user-guide/integrations/pulumi/index.md

```diff
@@ -5,20 +5,97 @@ description: >
   Use the Pulumi Infrastructure as Code framework with LocalStack
 ---
 
-<img src="pulumi-logo.svg" width="600px" alt="Pulumi logo">
+## Introduction
 
-## Overview
+Pulumi's SDK for infrastructure-as-code allows you to create, deploy, and manage AWS containers, serverless functions, and other infrastructure using popular programming languages. It supports a range of cloud providers, including AWS, Azure, Google Cloud, and Kubernetes.
 
-Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage AWS containers, serverless functions, and infrastructure using familiar programming languages.
-The endpoints configuration environment of Pulumi allows us to easily point Pulumi to LocalStack.
-This guide follows the instructions from Pulumi's [Get Started with Pulumi and AWS](https://www.pulumi.com/docs/get-started/aws/) guide, with additional explanations of how to make it work with LocalStack.
+LocalStack can integrate with Pulumi through the Pulumi configuration environment. There are two main methods to configure Pulumi for use with LocalStack:
 
+- Using the `pulumilocal` wrapper script which automatically configures service endpoints.
+- Manually setting up the service endpoints in your Pulumi configuration, which requires ongoing maintenance.
 
-## Quickstart
+This guide will show you how to set up local AWS resources using both the `pulumilocal` wrapper and manual configuration.
+
+## `pulumilocal` wrapper script
+
+`pulumilocal` is a wrapper for the `pulumi` command line interface, facilitating the use of Pulumi with LocalStack. When executing deployment commands like `pulumilocal ["up", "destroy", "preview", "cancel"]`, the script configures the Pulumi settings for LocalStack and runs the specified Pulumi command.
+
+The endpoints are set to point to the LocalStack API (`http://localhost:4566`). This setup simplifies the deployment of Pulumi stacks against LocalStack.
+
+### Configure the Local Backend
+
+Optionally, you can set environment variables to store state locally, avoiding cloud storage.
+
+{{< command >}}
+$ export PULUMI_CONFIG_PASSPHRASE=lsdevtest
+$ export PULUMI_BACKEND_URL=file://`pwd`/myproj
+{{< / command >}}
+
+{{< alert title="Note" >}}
+For further options please consult the official documentation on available  [environment variables](https://www.pulumi.com/docs/cli/environment-variables/) and [local backend](https://www.pulumi.com/docs/concepts/state/#local-filesystem).
+{{< /alert >}}
+
+### Create a new Pulumi project
+
+To start a new project, use these commands:
+
+{{< command >}}
+$ mkdir myproj
+$ pulumilocal new aws-typescript -y -s lsdev --cwd myproj
+{{< / command >}}
+
+{{< alert title="Note" >}}
+The `--cwd` option is unnecessary if you're already in the project directory.
+{{< /alert >}}
+
+### Install the `pulumilocal` wrapper script
```

Comment:
I'd move installation information to be listed after "Configure the local backend" section, otherwise the commands in the next section won't work.